javalib/src/main/scala/java/time/Preconditions.scala

```diff
@@ -0,0 +1,9 @@
+package java.time
+
+private[time] object Preconditions {
+  // Like scala.Predef.require, but throws a DateTimeException.
+  def require(requirement: Boolean, message: => Any): Unit = {
```

Comment:
Maybe it would be good to rename to avoid ambiguity. Could be `requireDateTime`.